.github/workflows/wheels.yml

```diff
@@ -46,14 +46,18 @@ jobs:
         run: |
           set -xe
           COMMIT_MSG=$(git log --no-merges -1)
-          echo "::set-output name=message::$COMMIT_MSG"
+          RUN="0"
```

Comment:
currently backporting this, though the maintenance branch may very well have the single line commit message check at this time